deployment/docker-compose/npmplus/.env

```
@@ -0,0 +1,71 @@
+APPSEC_MANAGER_PORT=8080
+
+APPSEC_MANAGER_ADMIN_USER=admin
+## Leave the password empty to have one generated and printed to the
+## manager's log on first start; change it after signing in.
+APPSEC_MANAGER_ADMIN_PASSWORD=
+APPSEC_MANAGER_DB_NAME=appsec_manager
+APPSEC_MANAGER_EVENT_RETENTION_DAYS=30
+
+## Postgres, used only by the manager.
+APPSEC_DB_USER=postgres
+APPSEC_DB_PASSWORD=pass
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPSEC_DB_PASSWORD key should go before the APPSEC_DB_USER key

(UnorderedKey)